allure-generator/src/main/javascript/components/testresult/TestResultView.js

```diff
@@ -1,70 +1,61 @@
 import './styles.scss';
 import {View} from 'backbone.marionette';
-import {on, regions, behavior} from '../../decorators';
-import pluginsRegistry from '../../util/pluginsRegistry';
+import {regions, behavior, className} from '../../decorators';
 import template from './TestResultView.hbs';
-import ExecutionView from '../execution/ExecutionView';
+import {Model} from 'backbone';
+import TestResultOverviewView from '../testresult-overview/TestResultOverviewView';
+import TestResultExecutionView from '../testresult-execution/TestResultExecutionView';
+import ErrorSplashView from '../error-splash/ErrorSplashView';
+import pluginsRegistry from '../../util/pluginsRegistry';
 
-const SEVERITY_ICONS = {
-    blocker: 'fa fa-exclamation-triangle',
-    critical: 'fa fa-exclamation',
-    normal: 'fa fa-file-o',
-    minor: 'fa fa-arrow-down',
-    trivial: 'fa fa-long-arrow-down'
-};
+const subViews = [
+    {id: '', name: 'testResult.overview.name', View: TestResultOverviewView},
+    {id: 'execution', name: 'testResult.execution.name', View: TestResultExecutionView}
```

Comment:
this PR is the base for future changes, execution tab is not finished yet, so as soon as it will be ready we will decide where we should display steps